packages/toolkit/package.json

```diff
@@ -50,12 +50,16 @@
   },
   "devDependencies": {
     "@arethetypeswrong/cli": "^0.13.5",
+    "@babel/core": "^7.24.8",
```

Comment:
:question: Why do we need Babel here?

Reply:
`mangleErrors.cjs` and `tsup.config.ts` use it.